src/wallet/signer.rs

```diff
@@ -129,7 +129,7 @@ impl From<Fingerprint> for SignerId {
 }
 
 /// Signing error
-#[derive(Debug, PartialEq, Eq, Clone)]
+#[derive(Debug)]
```

Comment:
Why are you removing PartialEq, Eq, Clone? Can't you make `hwi::error::Error` implement those?

Reply:
Unfortunately, no. `hwi::error::Error` contains `serde_json::error::Error` and `std::io::Error` which do not implement these traits. I could create a new implementation, but I don't think it would be useful (for `hwi::error::Error`). If these traits are absolutely necessary, then I think I could add an implementation.

Reply:
Maybe it's just easier to change hwi so that its error type implements those

Reply:
We could use `String` instead of `hwi::error::Error` here, or `String` inside hwi instead of `serde_json::error::Error` and `std::io::Error`